features/steps/core.py

```diff
@@ -66,6 +67,25 @@ def set_config(context, config_file):
             cf.write("version: {}".format(__version__))
 
 
+@when('we open the editor and enter')
+@when('we open the editor and enter "{text}"')
```

Comment:
Apparently you also have to add
`@when(u'we open the editor and enter ""')` to make {text} optional.

Reply:
I just added a "we open the editor and enter nothing" case, which takes advantage of the `text=""` default in the method signature.

Reply:
Apparently you can also make it optional by setting it to {text?} but only if the package `parse-type` is installed...
https://behave.readthedocs.io/en/latest/tutorial.html#python-step-implementations

Reply:
Fixed it.